### ISSUE TITLE
Feature Request: Disable Copy to system clipboard

### ISSUE THREAD

**Issue by kevintraver** [kyazdani42/nvim-tree.lua#789]
It would be nice to have the ability to disable yanking to the `+` register.

**Comment by kyazdani42** [comment]
not sure how to do this, maybe you could make a PR if you can :)

**Comment by kevintraver** [comment]
Could probably be something like this:

```
  if config.use_system_clipboard then
    vim.fn.setreg('+', content);
    vim.fn.setreg('"', content);
  end
```

I'm not exactly how to sure load the config options in `fs.lua`

**Comment by kyazdani42** [comment]
Could you write a little function of your own in your config and port it here ?

**Comment by lrnzlo** [comment]
Hi!
I've provide #1141 to solve this. If there's any question please ask!

**Comment by kyazdani42** [comment]
closed in #1141
